### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sman54815**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# online-bookstore
\ No newline at end of file
+# online-bookstore
```
Comment (mikhailfayez) [inline README.md:1]: what did u do ?

Comment (mikhailfayez) [reply README.md:1]: test

Comment (sman54815) [reply README.md:1]: ?

Comment (mikhailfayez) [reply README.md:1]: !

Comment (mikhailfayez) [reply README.md:1]: > ?

test

Comment (sman54815) [reply README.md:1]: @mikhailfayez

**Comment by sman54815** [comment]
test

**Comment by sman54815** [comment]
test

**Pull request closed without merging**